sos/report/plugins/seagate_ses.py

```diff
@@ -0,0 +1,59 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, IndependentPlugin
+
+
+class SeagateSES(Plugin, IndependentPlugin):
+
+    short_desc = 'Seagate SES status'
+    plugin_name = 'seagate_ses'
+    plugin_timeout = 600
+    profiles = ('system', 'storage', 'hardware',)
+    packages = ('fwdownloader_megaraid',)
```

Comment:
Where does this come from? I haven't been able to find it with typical Google-fu.


Is there a possiblity of a user have the `fwdownloader` utility without installing this specific package? If so, we'll want to add the `commands` tuple for an additional enablement trigger.